src/api/index.js

```diff
@@ -103,10 +103,29 @@ function putAnswer(experimentId, assignmentId, answer) {
   );
 }
 
+function exportList() {
+  return apiCall('/api/exports');
+}
+
+function exportCreate() {
+  return apiCall(`/api/exports`, {
+    method: 'POST',
+  });
+}
+
+function exportDownload(filename) {
+  const token = TokenService.get();
+  const url = apiUrl(`/api/exports/${filename}/download?jwt_token=${token}`);
```

Comment:
but... is not the `jwt_token` something included in all requests?
```
authorization:Bearer header.payload.signature
```
why should it explicitly sent?